format/Layout.md

```diff
@@ -89,8 +89,9 @@ maximum of 2^31 - 1 elements. We choose a signed int32 for a couple reasons:
 
 Any relative type can be nullable or non-nullable.
 
-Nullable arrays have a contiguous memory buffer, known as the validity (or
-null) bitmap, whose length is large enough to have 1 bit for each array slot.
+Nullable arrays have a contiguous memory buffer, known as the null (or
+validity) bitmap, whose length is large enough to have 1 bit for each array
+slot.
```

Comment:
I would propose that the null bitmap is always an multiple of 8 bytes in length. This simplifies some code to avoid having to manage partial word conditions.